pyproject.toml

```diff
@@ -45,7 +45,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
 
-lava-nc = { git = "https://github.com/lava-nc/lava.git", branch = "main", tag = "v0.3.0" }
+lava-nc = { git = "https://github.com/lava-nc/lava.git", branch = "main", develop = true}
```

Comment:
Should this be main rather than release/v0.4.0?

Reply:
Just wondering if we want the lava libs to fix support at releases...

Reply:
I needed to fix some things on lava side as well to fix some broken changes so had to point to the newest tip of lava to make the CI work. Before we release it, we should change the lava decency back to the newer tag of lava (v0.5.0)